### DISCUSSION TITLE
planet.pytest.org

### DISCUSSION THREAD

**Discussion by fschulze** [pytest-dev/pytest#9020] [General]
I was doing admin work on an old machine and noticed that planet.pytest.org ran there a while ago. Like all other pytest.org subdomains cloudflare.com is in front and pointing to another IP address which I don't recognize and which only shows the Apache default page. Should I just remove the planet subdomain from cloudflare? It most likely doesn't work for quite a while now.

**Comment by nicoddemus** [comment]
I think so, thanks @fschulze.

**Comment by fschulze** [comment]
I've removed it. It is easy enough to add back if needed.
